packages/block-editor/src/components/inserter/style.scss

```diff
@@ -214,8 +214,9 @@ $block-inserter-tabs-height: 44px;
 
 .block-editor-inserter__media-tabs-container,
 .block-editor-inserter__block-patterns-tabs-container {
+	flex-shrink: 0;
 	padding: $grid-unit-20;
-	height: 100%;
+	min-height: 100%;
```

Comment:
Thanks for fixing this! From what I tested `flex: 1` suffices and neither `flex-shrink: 0` or `min-height` is needed here. If I’m wrong about `flex-shrink`, I’d still prefer `flex: 1 0`.

Reply:
Sure, let's do that 👍